tests/runtimes/simple-bank/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+//! Simple bank runtime.
+use oasis_runtime_sdk::{self as sdk, core::common::version::Version, modules};
```

Comment:
```suggestion
use oasis_runtime_sdk::{self as sdk, core::common::version::Version, modules};
```
can you re-export `Version` from the sdk crate root? It's awful to type